homeassistant/components/sensor/const.py

```diff
@@ -139,6 +140,12 @@ class SensorDeviceClass(StrEnum):
     Unit of measurement: `ppm` (parts per million)
     """
 
+    CONCENTRATION = "concentration"
```

Comment:
This PR is changing our entity model. Before we can review this PR there needs to be approval in a discussion in our architecture repository.

https://github.com/home-assistant/architecture/discussions

https://developers.home-assistant.io/docs/core/entity#changing-the-entity-model

Also, please don't change the entity model in the same PR as changing an integration.

Reply:
I will create a follow up PR for the entity model changes after we discuss this integration. I have [committed and pushed a change](https://github.com/home-assistant/core/pull/124459/commits/40e9875ca84976800f07caaeaaf7f191fbe17918) that doesn't require the entity model changes and leaves pH as unitless, doesn't add CONCENTRATION, doesn't add TOTAL_DISSOLVED_SOLIDS and thus uses different types underneath for the new sensors/numbers.